airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/ConnectionReplicationTab.tsx

```diff
@@ -203,18 +203,20 @@ export const ConnectionReplicationTab: React.FC = () => {
                   dirty={dirty || schemaHasBeenRefreshed}
                 />
                 {status.editControlsVisible && (
-                  <EditControls
-                    isSubmitting={isSubmitting}
-                    submitDisabled={!isValid}
-                    dirty={dirty}
-                    resetForm={async () => {
-                      resetForm();
-                      discardRefreshedSchema();
-                    }}
-                    successMessage={saved && !dirty && <FormattedMessage id="form.changesSaved" />}
-                    errorMessage={getErrorMessage(isValid, dirty)}
-                    enableControls={schemaHasBeenRefreshed || dirty}
-                  />
+                  <div>
```

Comment:
Why does this need to be inside a div? should the div be part of EditControls?

Reply:
Sorry! Forgot to remove it after some local tests